gems/laa_multi_step_forms/app/helpers/laa_multi_step_forms/application_helper.rb

```diff
@@ -26,6 +26,12 @@ def app_environment
       "app-environment-#{ENV.fetch('ENV', 'local')}"
     end
 
+    def phase_name
+      return t('layouts.phase_banner.phase') if ENV.fetch('ENV', 'local') == 'production'
+
+      ENV.fetch('ENV', 'local')
```

Comment:
Now that we're on frontend v5, you'll want to capitalize this output as the tag helper won't do it for you.